src/pages/profile/myrecruit.tsx

```diff
@@ -1,130 +1,183 @@
-                      {item.name}
-                    </h4>
-                    <div>
-                      <p className='text-[13px] text-[#707070]'>
-                        {item.meeting_type} - 매주 {item.day} {item.time}
-                      </p>
-                    </div>
-                    <div className='flex justify-between items-center'>
-                      <div className='h-full flex'>
-                        {item.userGroup.map((user) => (
-                          <Image
-                            key={user.photoId}
-                            src={user.photoId}
-                            alt={user.nickname}
-                            width='0'
-                            height='0'
-                            className='w-6 h-6 bg-[#d9d9d9] border rounded-[50%] border-solid border-white'
-                          />
-                        ))}
-                      </div>
-                      <span className='flex justify-center items-center w-[2.8rem] h-5 bg-main text-[11px] text-white rounded-[3px]'>
-                        {item.recruitment_status ? '모집중' : '모집완료'}
+      {recruitStatus === 'success' && recruitData && (
+        <main className='flex flex-col gap-12 px-6 py-10  h-[calc(100%_-_8.5rem)]'>
+          {!userData?.isMine && userData?.bookshelfIsHidden ? (
+            <div className='h-full flex flex-col justify-center items-center '>
+              <h5 className='text-base font-medium text-[#333333]'>
+                비공개 계정입니다.
+              </h5>
+              <p className='text-[13px] text-[#707070]'>
+                이 계정은 확인할 수 없습니다.
+              </p>
+            </div>
+          ) : (
+            <>
+              {' '}
+              <section className='flex flex-col justify-center h-3/6'>
+                <h3 className='font-medium text-[17px] text-[#333333] mb-2'>
+                  운영중인 모임
+                </h3>
+                <div className='w-full h-36 flex flex-col overflow-scroll gap-3.5'>
+                  {leadingGroup && leadingGroup.length > 0 ? (
+                    leadingGroup.map((item) => (
+                      <article
+                        className='w-full h-full bg-white border flex flex-col gap-2 p-6 rounded-[10px] border-solid border-[#dfdfdf]'
+                        key={item.group_id}
+                      >
+                        <h4 className='text-[15px] text-[#333333] font-[bold]'>
+                          {item.name}
+                        </h4>
+                        <div>
+                          <p className='text-[13px] text-[#707070]'>
+                            {item.meeting_type} - 매주 {item.day} {item.time}
+                          </p>
+                        </div>
+                        <div className='flex justify-between items-center'>
+                          <div className='h-full flex'>
+                            {item.userGroup.map((user) => (
```

Comment:
map에서도 구조분해할당으로 사용할 수도 있을 것 같아요!
```suggestion
{item.userGroup.map(({photoId, photoUrl, nickname}) => (